app/views/evaluation_forms/_form.html.erb

```diff
@@ -5,6 +5,10 @@
       phase = evaluation_form.phase
       combo_box_default_value = "#{challenge.id}.#{phase.id}.#{phase.end_date.strftime("%m/%d/%Y")}"
     end
+
+    if evaluation_form.valid?
+      disabled = phase.end_date < Date.today
+    end
```

Comment:
you could make this a helper method in the EvaluationFormHelper instead and reuse it elsewhere:
```
def eval_form_disabled?(evaluation_form)
  evaluation_form.valid? && evaluation_form.phase.end_date < Date.today
end
```